src/lib/common/Kconfig

```diff
@@ -313,6 +332,16 @@ menuconfig HAVE_PIN_MUX
 
           If unsure, say Y.
 
+config MAINLOOP_FD
```

Comment:
Sounds better, done in the patch 'squash to: Remove mainloop stub implementations of Small Os'
If no other changes were requested in this PR I will just squash and merge it.